bin/stacks/api-stack.ts

```diff
@@ -377,12 +380,16 @@ export class APIStack extends cdk.Stack {
     });
     const enabled = switchResource.addResource('enabled');
 
-    /* add auth key */
+    /* add auth keys */
+    const tradingAPIKey = api.addApiKey('TradingAPIKey');
+    const devAPIKey = api.addApiKey('DevAPIKey');
     const apiAuthzKey = api.addApiKey('AuthzKey');
```

Comment:
I may have missed this but what is authz?

Reply:
It looks like it was used for the "synthetic switch" lambda, see this line:
`enabled.addMethod('GET', switchLambdaIntegration, { apiKeyRequired: true });`

I don't think the synthetic switch is still being used.